src/Core/maps/src/Core/IMap.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.Generic;
+using Microsoft.Maui.Devices.Sensors;
+
+namespace Microsoft.Maui.Maps
+{
+	/// <summary>
+	/// Represents a <see cref="IView"/> that displays a map.
+	/// </summary>
+	public interface IMap : IView
+	{
+		/// <summary>
+		/// The currently visible MapSpan of this Map.
+		/// </summary>
+		MapSpan? VisibleRegion { get; set; }
+
+		/// <summary>
+		/// Gets the display type of map that can be shown.
+		/// </summary>
+		MapType MapType { get; }
+
+		/// <summary>
+		/// Get whether the Map is showing the user's current location.
+		/// </summary>
+		bool IsShowingUser { get; }
+
+		/// <summary>
+		/// Get whether the Map is allowed to scroll.
+		/// </summary>
+		bool HasScrollEnabled { get; }
```

Comment:
Why are all of the boolean properties "Has" instead of "Is"?

Reply:
It was what we had before? Do we change on Core and Controls ?

Reply:
Done